EKF/ekf.h

```diff
@@ -385,6 +385,10 @@ class Ekf : public EstimatorInterface
 	Vector3f _delta_vel_bias_var_accum;		///< kahan summation algorithm accumulator for delta velocity bias variance
 	Vector3f _delta_angle_bias_var_accum;	///< kahan summation algorithm accumulator for delta angle bias variance
 
+	float _vert_pos_innov_ratio;		///< standard deviation of vertical position innovation
+	uint64_t _vert_pos_fuse_time_us;	///< last system time in usec vertical position measurement fuson was attempted
```

Comment:
Does it have to be the time of the last fusion **attempt**? If so, it would be great if the variable name would include that. If not, there is already `_time_last_hgt_fuse`.

Reply:
Unfortunately _time_last_hgt_fuse will only be set when there has an actual fusion event, whereas this use case requires all fusion attempts, regardless of whether the fusion was successful.

Reply:
What about `_vert_pos_fuse_attempt_time_us `?